src/components/CheckboxWithLabel.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View, TouchableOpacity} from 'react-native';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import Checkbox from './Checkbox';
+
+const propTypes = {
+    /** Component to display for label */
+    LabelComponent: PropTypes.func.isRequired,
+
+    /** Whether the checkbox is checked */
+    isChecked: PropTypes.bool.isRequired,
+
+    /** Called when the checkbox or label is pressed */
+    onPress: PropTypes.func.isRequired,
+
+    /** Container styles */
+    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+};
+
+const defaultProps = {
+    style: [],
+};
+
+const CheckboxWithLabel = ({
+    LabelComponent, isChecked, onPress, style,
```

Comment:
NAB, Should be moved to separate lines?

Reply:
Not sure we enforce this so if eslint doesn't mind I don't mind  😄